website/docs/add-secure-apps/flows-stages/stages/user_login/index.md

```diff
@@ -8,19 +8,18 @@ It can be used after `user_write` during an enrollment flow, or after a `passwor
 
 ## User login stage configuration options
 
-When creating or editing this stage in the UI of the Admin interface, you can set the following configuration options.
+When creating or editing this stage, you can set the following configuration options.
 
-**Name**: enter a descriptive name for the stage.
+### Session duration
 
-**Stage-specific settings**
-
-- **Session duration**: By default, the authentik session expires when you close your browser (_seconds=0_).
+By default, the authentik session expires when you close your browser (_seconds=0_).
```

Comment:
Because the Head3 is "Session settings", we need to leave the actual name of the setting, even if is seems obvious/redundant.

Reply:
Wait a minute, I am rethinking this now that I scroll down and see the other other settings ... yes, ok I agree that using the H3 to specify the setting name works. Though this would be inconsistent with other docs, were we bullet lists to show UI fields/settings....